packages/PHPStanStaticTypeMapper/TypeMapper/AccessoryNumericStringTypeMapper.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\PHPStanStaticTypeMapper\TypeMapper;
+
+use PhpParser\Node;
+use PhpParser\Node\Name;
+use PHPStan\PhpDocParser\Ast\Type\IdentifierTypeNode;
+use PHPStan\PhpDocParser\Ast\Type\TypeNode;
+use PHPStan\Type\Accessory\AccessoryNumericStringType;
+use PHPStan\Type\Type;
+use Rector\PHPStanStaticTypeMapper\Contract\TypeMapperInterface;
+use Rector\PHPStanStaticTypeMapper\Enum\TypeKind;
+
+/**
+ * @implements TypeMapperInterface<AccessoryNumericStringType>
+ */
+final class AccessoryNumericStringTypeMapper implements TypeMapperInterface
+{
+    /**
+     * @return class-string<Type>
+     */
+    public function getNodeClass(): string
+    {
+        return AccessoryNumericStringType::class;
+    }
+
+    /**
+     * @param AccessoryNumericStringType $type
+     */
+    public function mapToPHPStanPhpDocTypeNode(Type $type, TypeKind $typeKind): TypeNode
+    {
+        return new IdentifierTypeNode('string');
```

Comment:
Phpstan also supports `numeric-string`.. not sure this would be correct to use here though

See https://phpstan.org/developing-extensions/type-system

Reply:
The PHP doc is also for PHPStorm, that understands only `string` type.